packages/storybook-test/stories/skip-link/skip-link-stories.tsx

```diff
@@ -0,0 +1,247 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Heading } from '../../../components-react/heading-react/src/css';
+import { Paragraph } from '../../../components-react/paragraph-react/src/css';
+import packageJSON from '../../../components-react/skip-link-react/package.json';
+import { SkipLink } from '../../../components-react/skip-link-react/src/css';
+import { ExampleBodyText } from '../../src/ExampleBodyText';
+import '../../../components-css/skip-link-css/src/test.scss';
+
+const meta = {
+  argTypes: {
+    children: { table: { category: 'API' }, type: 'string' },
+    href: { table: { category: 'API' } },
+  },
+  component: SkipLink,
+  parameters: {
+    docs: {
+      description: {
+        component: 'Component beschrijving',
+      },
+    },
+    externalLinks: [
+      {
+        name: 'Open in Figma',
+        url: 'https://www.figma.com/design/shhwGcqPLi2CapK0P1zz8O/NLDS---Voorbeeld---Bibliotheek?node-id=2217-10122',
+      },
+      {
+        name: 'Open op GitHub',
+        url: packageJSON.homepage,
+      },
+    ],
+  },
+  title: 'Componenten/Skip Link',
+} satisfies Meta<typeof SkipLink>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  name: 'Skip Link',
+  args: {
+    children: 'Naar de inhoud',
+    href: '#main-story-1',
+  },
+  globals: {
+    dir: 'ltr',
+    lang: 'nl',
+    title: 'Contact',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: `Een kleine webpagina: naam van de organisatie, hoofdnavigatie met 3 links, een Heading, en een alinea tekst met daarin een link.
+
+Je kunt met \`Tab\` naar de Skip Link die eerst niet zichtbaar is. Daarmee kun je links in de navigatie overslaan. De volgende tab-stop is de link in de alinea.`,
+      },
+    },
+    status: { type: [] },
+  },
+  render(props) {
+    return (
+      <ExampleBodyText>
+        <SkipLink {...props} />
+        <header>
+          <Paragraph>
+            <bdi>Example Inc.</bdi>
+          </Paragraph>
+        </header>
+        <nav>
+          <ul>
+            <li>
+              <a href="https://example.com/">Homepage</a>
+            </li>
+            <li>
+              <a href="https://example.com/">Over ons</a>
+            </li>
+            <li>
+              <a href="https://example.com/">Contact</a>
+            </li>
+          </ul>
+        </nav>
+        <main id={props.href?.substring(1)}>
+          <Heading level={1}>Contact</Heading>
+          <Paragraph>
+            Je kunt contact met ons opnemen via ons e-mailadres:{' '}
+            <a href="mailto:hello@example.com">hello@example.com</a>.
+          </Paragraph>
+        </main>
+      </ExampleBodyText>
+    );
+  },
+};
+
+export const FocusedSkipLink: Story = {
+  name: 'Skip Link (zichtbaar)',
+  args: {
+    ...Default.args,
```

Comment:
Dit gaat een beetje in tegen alle stories statisch en expliciet hebben. We doen het ook nergens anders…